python/ray/train/tests/test_lightning_trainer.py

```diff
@@ -71,11 +71,13 @@ def test_trainer_with_native_dataloader(
     batch_size = 8
     num_workers = 2
     dataset_size = 256
+    ckpt_dirpath = "lightning/ckpts/dir"
```

Comment:
Should we test all the possibilities: default, with a relative ckpt_dirpath, and with an absolute ckpt_dirpath?

Also, let's add the dirpath to the tuning lightning release test.

Reply:
Done